### PR TITLE
feat(Skeleton): consumed Penta tokens

### PR DESCRIPTION
**Pull request by thatblindgeye**
Work towards #5738

### CODE REVIEW

**Review by andrew-ronaldson — APPROVED** [review]
LGTM, thanks

File: src/patternfly/components/Skeleton/skeleton.scss [hunk 1]
```diff
@@ -13,9 +14,9 @@
 
   // After
   --#{$skeleton}--after--LinearGradientAngle: 90deg;
-  --#{$skeleton}--after--LinearGradientColorStop1: var(--#{$pf-global}--palette--black-150);
+  --#{$skeleton}--after--LinearGradientColorStop1: var(--pf-t--global--background--color--secondary--default);
   --#{$skeleton}--after--LinearGradientColorStop2: #ededed;
```
Comment (srambach) [inline src/patternfly/components/Skeleton/skeleton.scss:18]: Let's use a token here... maybe `--pf-t--global--color--nonstatus--gray--default` ?

**Comment by thatblindgeye** [comment]
@srambach 


https://github.com/patternfly/patternfly/assets/70952936/113e6ede-6c83-4ea6-9464-e06f3d685b7b

**Comment by thatblindgeye** [comment]
https://github.com/patternfly/patternfly/assets/70952936/3fc64ccf-f3a5-4f02-8f7a-875cc2137bb0

**Review by srambach — APPROVED** [review]
Fabulous! 🦴

**Pull request merged**